patches/chromium/build_add_public_config_simdutf_config.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Mon, 9 Dec 2024 11:22:20 -0600
+Subject: build: add public config simdutf_config
+
+consumed by electron_node
```

Comment:
This should have a better description explaining why it is necessary

Reply:
Clarified in c48342b9ce514d5fb47e8ccb0d2ecad049216ed0